.github/workflows/build-x86-image.yaml

```
@@ -1064,6 +1153,7 @@ jobs:
       - build-centos-compile
       - k8s-conformance-e2e
       - k8s-netpol-e2e
+      - k8s-netpol-legacy-e2e
```

```suggestion
      # - k8s-netpol-legacy-e2e
```

Some e2e cases failed, we should comment this until the failure is resolved.

---

ok